pkgs/matcher/lib/src/map_matchers.dart

```diff
@@ -15,6 +15,7 @@ class _ContainsValue extends Matcher {
 
   @override
   bool matches(Object? item, Map matchState) =>
+      // ignore: avoid_dynamic_calls
       (item as dynamic).containsValue(_value);
```

Comment:
I am curious why we can't cast this to a Map?

Reply:
I may have been worried about non-map types with `operator []` and `containsKey` or `containsValue`, but thinking about it now that seems like an unlikely use case. I'll change this and `containsValue` to be `FeatureMatcher<Map>`